srcpkgs/gnuradio/template

```diff
@@ -1,32 +1,31 @@
 # Template file for 'gnuradio'
 pkgname=gnuradio
-version=3.7.13.5
-revision=2
+version=3.8.0.0
+revision=1
 build_style=cmake
-nocross="relies on volk, which is not complete for arm"
 conf_files="/etc/gnuradio/conf.d/*"
 configure_args="-DENABLE_INTERNAL_VOLK=OFF"
 pycompile_module="gnuradio grc_gnuradio pmt"
 hostmakedepends="pkg-config doxygen swig python-cheetah python-lxml
```

Comment:
Please remove `python-cheetah`, `python-lxml`, `python-numpy`, `python-Sphinx` and replace them with these: `python3-Mako python3-Sphinx python3-cairo python3-numpy python3-yaml`